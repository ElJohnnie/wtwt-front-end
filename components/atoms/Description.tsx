import styles from 'styles\Description.modules.css'

interface DescriptionProps{
    textDescription: string,
}

export default function Description(props: DescriptionProps){
    return <div>
        <p></p>
    </div>
}