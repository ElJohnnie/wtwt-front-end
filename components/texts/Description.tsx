import React from 'react';
import { DescriptionProps } from '../../types';

export default function Description(props: DescriptionProps) {
  const { text } = props;

  return <div className={`text-center text-pink-600`}>{text}</div>;
}
