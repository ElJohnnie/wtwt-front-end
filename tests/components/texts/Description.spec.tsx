import React from 'react';
import Description from '../../../components/texts/Description';
import '@testing-library/jest-dom';
import { renderWithOutContextProvider } from '../../../utils/tests/helpers';

describe('Description', () => {
  it('renderizar o component Description', () => {
    const { container, getByText } = renderWithOutContextProvider(
      <Description text="batatinha" />,
    );

    const receipt = getByText('batatinha');
    expect(receipt).toBeInTheDocument();
    expect(container).toBeTruthy();
    expect(container).toMatchSnapshot();
  });
});
