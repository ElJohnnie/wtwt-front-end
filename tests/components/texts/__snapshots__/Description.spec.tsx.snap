// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description renderizar o component Description 1`] = `
<div>
  <div
    class="text-center text-pink-600"
  >
    batatinha
  </div>
</div>
`;
